verification/curator-service/test/sources.test.ts

```diff
@@ -16,29 +17,103 @@ afterAll(() => {
     return mongoose.disconnect();
 });
 
+beforeEach(() => {
+    return Source.deleteMany({});
+});
+
 describe('GET', () => {
-    it('list should return 200', (done) => {
-        request(app).get('/api/sources').expect(200, done);
+    it('list should return 200', async () => {
+        const source = new Source({
+            name: 'test-source',
+            origin: { url: 'http://foo.bar' },
+        });
+        const saved = await source.save();
```

Comment:
mongoose updates the saved document (e.g. you can simply use the `source` object)

Reply:
nice, thanks!